arrow-buffer/src/bigint.rs

```diff
@@ -455,6 +504,39 @@ mod tests {
                 expected
             ),
         }
+
+        // Exponentiation
+        for exp in vec![0, 1, u32::MAX].into_iter() {
```

Comment:
```suggestion
        for exp in vec![0, 1,3, 8, u32::MAX].into_iter() {
```

Just to get a bit more coverage